Corona-Warn-App/src/deviceForTesters/res/layout/fragment_settings_crash_report_details.xml

```diff
@@ -1,39 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".test.crash.ui.SettingsCrashReportDetailsFragment">
+    xmlns:tools="http://schemas.android.com/tools">
 
+    <data>
 
-    <ScrollView
+        <variable
+            name="settingsCrashReportViewModel"
```

Comment:
Don't pass the whole viewmodel, pass a state object for the UI.
While in this simple case it will not be too problematic, starting to do this is what got us into the nested-nested-nested-viewmodel issues we had with the `HomeFragment` etc.

Reply:
adjusted